great_expectations/checkpoint/actions.py

```diff
@@ -161,7 +163,7 @@ class (for open source Great Expectations) or the GeCloudIdentifier (from Great
         """  # noqa: E501
 
     # NOTE: To be promoted to 'run' after V1 development (JIRA: V1-271)
-    def v1_run(self, checkpoint_result: CheckpointResult) -> None:
+    def v1_run(self, checkpoint_result: CheckpointResult) -> str | dict:
```

Comment:
Not directly related to this PR but I noticed the `_run` method is annotated as public which is strange since it's a `_` method. Could you remove that either in this or a followup PR?